clang/test/CodeGenHLSL/builtins/normalize.hlsl

```diff
@@ -0,0 +1,73 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple \
+// RUN:   dxil-pc-shadermodel6.3-library %s -fnative-half-type \

```

Comment:
Shall this also check the spv equivalent of the intrinsic is emitted then the target is spv?